back/src/main/java/sellerlee/back/article/presentation/ArticleController.java

```diff
@@ -67,6 +67,14 @@ public ArticleController(ArticleService articleService, ArticleViewService artic
         return ResponseEntity.ok(responses);
     }
 
+    @GetMapping(params = "tradeState")
+    public ResponseEntity<List<ArticleCardResponse>> showByTradeState(
+            @RequestParam String tradeState, @LoginMember Member loginMember) {
+        List<ArticleCardResponse> responses = articleViewService.showByTradeState(loginMember,
+                tradeState);
+        return ResponseEntity.ok(responses);
+    }
+
```

Comment:
params는 의도적으로 명시 해놓은 것인가요?
@requestParam 변수명과 같아서 알아서 매핑 되는 걸로 알아서용

```suggestion
    @GetMapping
    public ResponseEntity<List<ArticleCardResponse>> showByTradeState(
            @RequestParam String tradeState, @LoginMember Member loginMember) {
        List<ArticleCardResponse> responses = articleViewService.showByTradeState(loginMember,
                tradeState);
        return ResponseEntity.ok(responses);
    }

```

Reply:
> 생각해보면 이 코드가 나온 배경은 이미 /articles에 GET 메소드가 매핑 되어있기 때문이었을지도 모르겠습니다.
하나의 요청을 받을땐 명시하지 않아도 괜찮지만, 그렇지 않은 상황에선 @GetMapping 어노테이션에 params를 사용하여, URI와 method는 동일하고 params만 다른 요청을 처리할 수 있습니다.

해당 부분 혹시나해서 지워보고 다시 실험해봤는데요!
두 개 이상의 모호한 맵핑이 있을때 (현재는 `GET /articles` 요청이 3개가 있습니다.)
bean 생성에 실패하는 것으로 확인했습니다.

Reply:
엇 저두 포스트맨으로 실험해본거였는데 응답 잘 오더라구요 다시 살펴봐야겠네요

Reply:
컨트롤러 빈 생성 실패한다는건가요? 왜 저는 잘 돌아가죠 ㅠㅠㅠ

Reply:
2개는 명시 되어있고 1개는 명시 되어 있지 않은 상황이라 문제가 안생기는것 같아요.

Reply:
아 해당 내용 확인했습니다  굿굿